### ISSUE TITLE
Could not invoke FirestackAnalytics.logEventWithName

### ISSUE THREAD

**Issue by Sh3rawi** [fullstackreact/react-native-firestack#110]
Hello,
On Android 4.2.2 (and 4.2.1, 4.1.2 as well), whenever you use firestack.analytics.logEventWithName the app crashes with this stack trace:
``` 
Fatal Exception: java.lang.RuntimeException: Could not invoke FirestackAnalytics.logEventWithName
       at com.facebook.react.bridge.BaseJavaModule$JavaMethod.invoke(BaseJavaModule.java:332)
       at com.facebook.react.bridge.NativeModuleRegistry$ModuleDefinition.call(NativeModuleRegistry.java:187)
       at com.facebook.react.bridge.NativeModuleRegistry.call(NativeModuleRegistry.java:62)
       at com.facebook.react.bridge.CatalystInstanceImpl$NativeModulesReactCallback.call(CatalystInstanceImpl.java:432)
       at com.facebook.react.bridge.queue.NativeRunnableDeprecated.run(NativeRunnableDeprecated.java)
       at android.os.Handler.handleCallback(Handler.java:725)
       at android.os.Handler.dispatchMessage(Handler.java:92)
       at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:31)
       at android.os.Looper.loop(Looper.java:153)
       at com.facebook.react.bridge.queue.MessageQueueThreadImpl$3.run(MessageQueueThreadImpl.java:196)
       at java.lang.Thread.run(Thread.java:838)
Caused by java.lang.reflect.InvocationTargetException
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:511)
       at com.facebook.react.bridge.BaseJavaModule$JavaMethod.invoke(BaseJavaModule.java:319)
       at com.facebook.react.bridge.NativeModuleRegistry$ModuleDefinition.call(NativeModuleRegistry.java:187)
       at com.facebook.react.bridge.NativeModuleRegistry.call(NativeModuleRegistry.java:62)
       at com.facebook.react.bridge.CatalystInstanceImpl$NativeModulesReactCallback.call(CatalystInstanceImpl.java:432)
       at com.facebook.react.bridge.queue.NativeRunnableDeprecated.run(NativeRunnableDeprecated.java)
       at android.os.Handler.handleCallback(Handler.java:725)
       at android.os.Handler.dispatchMessage(Handler.java:92)
       at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:31)
       at android.os.Looper.loop(Looper.java:153)
       at com.facebook.react.bridge.queue.MessageQueueThreadImpl$3.run(MessageQueueThreadImpl.java:196)
       at java.lang.Thread.run(Thread.java:838)
Caused by java.lang.NoSuchMethodError: android.os.Bundle.getBinder
       at io.fullstack.firestack.FirestackAnalyticsModule.makeEventBundle(FirestackAnalytics.java:220)
       at io.fullstack.firestack.FirestackAnalyticsModule.logEventWithName(FirestackAnalytics.java:56)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:511)
       at com.facebook.react.bridge.BaseJavaModule$JavaMethod.invoke(BaseJavaModule.java:319)
       at com.facebook.react.bridge.NativeModuleRegistry$ModuleDefinition.call(NativeModuleRegistry.java:187)
       at com.facebook.react.bridge.NativeModuleRegistry.call(NativeModuleRegistry.java:62)
       at com.facebook.react.bridge.CatalystInstanceImpl$NativeModulesReactCallback.call(CatalystInstanceImpl.java:432)
       at com.facebook.react.bridge.queue.NativeRunnableDeprecated.run(NativeRunnableDeprecated.java)
       at android.os.Handler.handleCallback(Handler.java:725)
       at android.os.Handler.dispatchMessage(Handler.java:92)
       at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:31)
       at android.os.Looper.loop(Looper.java:153)
       at com.facebook.react.bridge.queue.MessageQueueThreadImpl$3.run(MessageQueueThreadImpl.java:196)
       at java.lang.Thread.run(Thread.java:838)
```

As getBinder was added in API 18 (our minSdkVersion is 16). Any specific reason for using getBinder instead of getBundle ?

**Comment by auser** [comment]
Does it work if you change it to `getBundle()`? I don't recall specifically _why_ I used `getBinder()`, TBH

**Comment by Sh3rawi** [comment]
Yup it works just fine

**Comment by Sh3rawi** [comment]
Here is a PR to fix it: #111
